frontend/server/src/Controllers/User.php

```diff
@@ -4678,36 +4678,29 @@ public static function getVerificationParentalTokenDetailsForTypeScript(
             throw new \OmegaUp\Exceptions\UnauthorizedException();
         }
 
-            $user = \OmegaUp\DAO\Users::getByPK($userId);
+        $user = \OmegaUp\DAO\Users::getByPK($userId);
         if (is_null($user)) {
             throw new \OmegaUp\Exceptions\NotFoundException('userNotExist');
         }
 
-        try {
             $user->parent_email_id = $r->user->main_email_id;
             $user->parent_verified = true;
             $user->parental_verification_token = null;
 
             \OmegaUp\DAO\Users::update($user);
-            $parentalVerificationToken = true;
-        } catch (\OmegaUp\Exceptions\ApiException $e) {
-            self::$log->error(
-                "Unable to save parental token verification: $e"
-            );
-        }
 
-        return [
+            return [
             'templateProperties' => [
                 'payload' => [
-                    'parentalVerificationToken' => $parentalVerificationToken,
+                    'hasParentalVerificationToken' => $hasParentalVerificationToken,
                 ],
                 'title' => new \OmegaUp\TranslationString(
                     'omegaupTitleParentalVerificationToken'
                 ),
             ],
             'entrypoint' => 'user_verification_parental_token',
-        ];
+            ];
     }
 }
 
-\OmegaUp\Controllers\User::$urlHelper = new \OmegaUp\UrlHelper();
+    \OmegaUp\Controllers\User::$urlHelper = new \OmegaUp\UrlHelper();
```

Comment:
```suggestion
\OmegaUp\Controllers\User::$urlHelper = new \OmegaUp\UrlHelper();
```

this doesn't seem related

Reply:
Ig my editor does these linting error or when i run /stuff/lint.sh. I'll undo all lint changes asap